drivers/platform/maxim/max32650/maxim_spi.c

```diff
@@ -169,6 +169,7 @@ int32_t max_spi_init(struct no_os_spi_desc **desc,
 	descriptor->mode = param->mode;
 	descriptor->bit_order = param->bit_order;
 	descriptor->platform_ops = &max_spi_ops;
+	descriptor->extra = eparam;
```

Comment:
i think all the changes in this commit should be distributed to each of the next commits, corresponding to each MAX part.